Doc/library/datetime.rst

```diff
@@ -283,15 +283,13 @@ Supported operations:
 | ``abs(t)``                     | equivalent to +\ *t* when ``t.days >= 0``, and|
 |                                | to -*t* when ``t.days < 0``. (2)              |
 +--------------------------------+-----------------------------------------------+
-| ``str(t)``                     | Returns a string in the form                  |
-|                                | ``[D day[s], ][H]H:MM:SS[.UUUUUU]``, where D  |
-|                                | is negative for negative ``t``. (5)           |
+| ``str(t)``                     | Returns a string in the form                  | |                                | ``[D day[s], ][H]H:MM:SS[.UUUUUU]``, where D  | |                                | is negative for negative ``t``. (5)           |
```

Comment:
Ah, crap. This was an illusion created by Vim's soft-wrapping. `git add -p` too broke the lines weirdly so I wasn't able to catch it. Am running `make html` now to verify the output. Sorry about missing this.